### ISSUE TITLE
Add option to specify error level :bar_chart: 

### ISSUE THREAD

**Issue by jamacku** [redhat-plumbers-in-action/differential-shellcheck#75]
### Type of issue

Feature Request

### Description

It would make sense to allow users to specify minimal error level that should be reported by differential-shellcheck

For example when user is interested only in errors it would look like:

```yml
  - name: Differential ShellCheck
    uses: redhat-plumbers-in-action/differential-shellcheck@latest
      with:
        level: error
        token: ${{ secrets.GITHUB_TOKEN }}
```

### Describe the solution you'd like

`csgrep` has option `--event` that could be used to implement this feature:

```
--event arg
              defect matches if its key event matches the given regex (each defect has exactly one key event, which determines its location in the code)
```

I woud expect something like this:

```yml
error:    '^error'
warning:  '^warning|^error'
note:     '^note|^warning|^error' <-- default
```

**Comment by jamacku** [comment]
ShellCheck has option for this:

> #### -S | --severity
>
> Specify minimum severity of errors to consider. Valid values in order of severity are error, warning, info and style. The default is style.

**Comment by lzaoral** [comment]
Note, that the `style` and `note` severity levels correspond to the `note` diagnostic messages when ShellCheck outputs in GCC format: https://github.com/koalaman/shellcheck/blob/4e703e5c61c6366bfd486d728bc624025e344e68/src/ShellCheck/Formatter/GCC.hs#L58-L61
